spec/filters/aggregate_spec.rb

```diff
@@ -0,0 +1,203 @@
+# encoding: utf-8
+require "logstash/devutils/rspec/spec_helper"
+require "logstash/filters/aggregate"
+
+describe LogStash::Filters::Aggregate do
+	def event(data = {})
```

Comment:
As you wish, naming is hard @fbaligand jejjje! usually the generic ones are names spec_helper.rb